worlds/sm64ex/__init__.py

```diff
@@ -3,7 +3,7 @@
 import json
 from .Items import item_table, action_item_table, cannon_item_table, SM64Item
 from .Locations import location_table, SM64Location
-from .Options import SM64Options
+from .Options import SM64Options, sm64_options_groups
```

Comment:
```suggestion
from .Options import sm64_options_groups, SM64Options 
```
nitpick: Import order should match the rest of the file where it is variables followed by classes.